content/en/error_tracking/sentry_sdk.md

```diff
@@ -0,0 +1,430 @@
+| JavaScript | `@sentry/node@9.13.0`<br>`@sentry/browser@9.13.0` |
+| Python     | `sentry-sdk==2.26.1`                              |
+| Java       | `io.sentry:sentry:8.6.0`                          |
+| .NET       | `Sentry 5.5.1`                                    |
+| Go         | `sentry-go v0.32.0`                               |
+| Ruby       | `sentry-ruby 5.23.0`                              |
+
+## Migrate to the recommended setup
+
+To get the most out of Error Tracking, it is recommended to eventually migrate to the Datadog SDK and/or Agent-based setups. See [Backend Error Tracking][7] and [Frontend Error Tracking][8] on how to configure this.
+
+The setup using the Sentry SDK can be used simultaneously with the recommended setup. Errors may be reported twice.
+
+## Send events to both Sentry and Datadog
+Events can be sent to both Sentry (or any other Sentry-compatible backend) and Datadog. This allows you to start using Datadog while also keeping your current solution. There are several ways to achieve this.
+
+### From the Sentry SDK
+Sentry SDKs can be configured to send events to multiple DSNs at once. On most Sentry SDKs, you can override the default transport to achieve this.
+
+{{< tabs >}}
+
+{{% tab "JavaScript" %}}
+{{< code-block lang="javascript" >}}
+// Change to import from "@sentry/react", "@sentry/nextjs", etc. as needed
+import * as Sentry from "@sentry/browser";
+import { makeFetchTransport } from "@sentry/browser"; // import { makeNodeTransport } from "@sentry/node" for Node.js
+import { makeMultiplexedTransport } from "@sentry/core";
+
+const sentryDsn = '<SENTRY_DSN>';
+const datadogDsn = '<DATADOG_DSN>';
+
+Sentry.init({
+  dsn: sentryDsn,
+  transport: makeMultiplexedTransport(makeFetchTransport, () => [sentryDsn, datadogDsn]),
+  // ...
+});
+Sentry.setTag('service', 'my-app');
+{{< /code-block >}}
+{{% /tab %}}
+
+{{% tab "Python" %}}
+- Copy the following function into your code:
+{{< code-block lang="python" >}}
+from sentry_sdk.transport import Transport, make_transport
+def make_multi_transport(dsns):
+    class MultiTransport(Transport):
+        def __init__(self, options):
+            super().__init__(options)
+            self.transports = [
+                make_transport({**options, "dsn": dsn, "transport": None}) for dsn in dsns
+            ]
+        def capture_envelope(self, *args, **kwargs):
+            for transport in self.transports:
+                transport.capture_envelope(*args, **kwargs)
+        def flush(self, *args, **kwargs):
+            for transport in self.transports:
+                transport.flush(*args, **kwargs)
+        def kill(self):
+            for transport in self.transports:
+                transport.kill()
+        def record_lost_event(self, *args, **kwargs):
+            for transport in self.transports:
+                transport.record_lost_event(*args, **kwargs)
+    return MultiTransport
+{{< /code-block >}}
+
+- Use as follows:
```

Comment:
```suggestion
2. Use as follows:
```
I find this phrasing a little awkward and vague, and we use it a few times. I'm wondering if there's more context we can give - for example, why these code snippets are separated out and where they should go?